### DISCUSSION TITLE
2022-01-24 Core team update

### DISCUSSION THREAD

**Discussion by mm-near** [near/community#188] [Core]
# TL;DR

* debugging issues with OOM related to the new release (discovered couple issues - now brainstorming on how to fix them best)
* more storage / rocksdb analysis & planned improvements

# Done

@mzhangmzz - debugging issues related to the release
@mzhangmzz - Fix a bug in block sync to make it request three blocks at a time https://github.com/near/nearcore/pull/6160
@mzhangmzz - Fix for chunk catchup sending too many messages https://github.com/near/nearcore/pull/6148
@mzhangmzz - Fix error messages during chunk requesting https://github.com/near/nearcore/issues/6100

@Longarithm - chunk level cache for used nodes
@Longarithm - rocksdb improvements brainstorming

@mm-near - debugging OOM issues related to the recent release
@mm-near - rocksdb optimizations analysis

@jc-near joined.
